roseau/load_flow/models/transformers/transformers.py

```diff
@@ -16,19 +16,22 @@
 class Transformer(AbstractBranch):
     """A generic transformer model.
 
-    The model parameters and windings type are defined in the ``parameters``.
+    The model parameters are defined in the ``parameters``.
     """
 
     branch_type = BranchType.TRANSFORMER
 
-    allowed_phases = frozenset({"abc", "abcn"})  # Only these for now
+    allowed_phases = Bus.allowed_phases
     """The allowed phases for a transformer are:
 
-    - P-P-P or P-P-P-N: ``"abc"``, ``"abcn"``
-
-    .. note::
-        Only 3-phase transformers are currently supported.
+    - P-P-P or P-P-P-N: ``"abc"``, ``"abcn"`` (tri-phase transformer)
+    - P-P or P-N: ``"ab"``, ``"bc"``, ``"ca"``, ``"an"``, ``"bn"``, ``"cn"`` (single-phase transformer or primary of
+    split-phase transformer)
```

Comment:
Keep the indentation correct for a markdown list
```suggestion
    - P-P or P-N: ``"ab"``, ``"bc"``, ``"ca"``, ``"an"``, ``"bn"``, ``"cn"`` (single-phase
      transformer or primary of split-phase transformer)
```